tests/EdgeToEdge/APIRoutes/AddressChangeControllerTest.php

```diff
@@ -186,12 +186,20 @@ private function makeAddressChange( FunFunFactory $factory, AddressType $address
 		return $addressChange;
 	}
 
+	/**
+	 * @return string[]
```

Comment:
```suggestion
	 * @return array{string,string}
```

This is meant as a tuple.